part2/bowling/bowling.rb

```diff
@@ -0,0 +1,42 @@
+class Game
+	def initialize
+		@score = []
+		@frames_arr = []
+	end
+
+	def score
+		# @frames_arr = @score.each_slice(2).to_a
+		frames
+		@score = @frames_arr.sum
+		# @score.sum
```

Comment:
Great start!

In order to think about the `spare` logic, have a think about how you could model sequential rolls.